app/scripts/e2e-run-session.sh

```
@@ -0,0 +1,400 @@
+# binary has no co-located libcef.so / libcef.dll. CEF lives in the
+# vendored-tauri-cli cache that `ensure-tauri-cli.sh` pinned via CEF_PATH:
+#   $CEF_PATH/<cef-version>/cef_<os>_<arch>/{libcef.so,libcef.dll,Resources,…}
+#
+# We find the only versioned directory under CEF_PATH and prepend its CEF
+# dist dir to LD_LIBRARY_PATH (Linux) / PATH (Windows). On macOS this is a
+# no-op — the .app bundle already self-resolves.
+# ------------------------------------------------------------------------------
+CEF_PATH="${CEF_PATH:-$HOME/Library/Caches/tauri-cef}"
+case "$OS" in
+  Linux)
+    CEF_DIST_DIR="$(ls -d "$CEF_PATH"/*/cef_linux_* 2>/dev/null | head -1)"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)